pkg/kubelet/types/constants.go

```diff
@@ -21,3 +21,21 @@ const (
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
 )
+
+// HairpinMode denotes how the kubelet should configure networking to handle
+// hairpin packets.
+type HairpinMode string
+
+// Enum settings for different ways to handle hairpin packets.
+const (
+	// Set the hairpin flag on the veth of containers in the respective
+	// container runtime.
+	HairpinFlag = "flag"
```

Comment:
how about these names for the options:
i) hairpin-veth, ii) promiscous-bridge, iii) none
or maybe something that conveys the essence more closely ?